modules/apps/29-fee/keeper/keeper.go

```diff
@@ -85,6 +85,19 @@ func (k Keeper) EscrowAccountHasBalance(ctx sdk.Context, coins sdk.Coins) bool {
 	return true
 }
 
+// lockFeeModule sets a flag to determine if fee handling logic should run for the given channel
```

Comment:
Should we add an `unlockFeeModule` helper fn also or is the idea to do that in a follow-up PR?

I'm also unclear as to what the actual process looks like for unlocking. What is your idea for that (sorry if this is written down somewhere I couldn't find mention in the ADR/issues) :) 

Once we have the unlock flow I guess it would make sense to also have a test case for:

1. Lock fee module
2. Try to incentivize packet (doesn't work)
3. Unlock fee module
4. Try to incentivize packet (works)